include/seqan3/argument_parser/detail/format_base.hpp

```diff
@@ -466,21 +431,21 @@ class format_help_base : public format_base
     bool show_advanced_options{true};
 
 private:
-    /*!\brief Checks whether the information annotated with `spec` should be added to the help page.
-     * \param[in] spec The option specification.
-     * \returns `true` if the information annotated with `spec` should be added to the help page, `false` otherwise.
+    /*!\brief Adds a function object to parser_set_up_calls **if** the annotation in `spec` does not prevent it.
+     * \param[in] fun The function object that, if added to parser_set_up_calls, prints information to the help page.
```

Comment:
```suggestion
     * \param[in] print_callback The invocable that, if added to `parser_set_up_calls`, prints information to the help page.
```

Reply:
The name `print_callback` does not tell me much. I don't know what you are expressing by that?

Reply:
@rrahn ping

Reply:
> In computer programming, a callback, also known as a "call-after"[1] function, is any executable code that is passed as an argument to other code; that other code is expected to call back (execute) the argument at a given time.

So that's why callback tells me what it is. Since it is documented as invocable it would be fine for me to call it printer.